core/tests/protractor_utils/ExplorationEditorPage.js

```diff
@@ -173,6 +175,13 @@ var ExplorationEditorPage = function() {
     navigateToTranslationTabButton.click();
     waitFor.pageToFullyLoad();
   };
+  this.navigateToTranslationTabFeedback = function() {
+    waitFor.elementToBeClickable(
+      navigateToTranslationTabButtonFeedback,
+      'Translation tab is not clickable');
```

Comment:
Are we expecting the translation tab button to be clickable here?

Reply:
Thanks for pointing out.
Changing it to 'Feedback sub tab of translation tab is not clickable'.

Reply:
Changed it to `'Feedback tab of translation page is not clickable'`